swagger.yml

```
@@ -224,17 +224,17 @@ paths:
         default:
           $ref: '#/responses/error'
       description: >-
-        [Beta] Returns all environment variables for an account or site. An
-        account corresponds to a team in the Netlify UI. To use this endpoint,
-        opt in to the beta environment variable experience using the Netlify UI.
+        Returns all environment variables for an account or site. An
+        account corresponds to a team in the Netlify UI. To use this endpoint, your 
+        site must no longer be using the classic environment experience. Migrate now with the Netlify UI.
```

Sorry, I bet this typo was my fault, I forgot to include the word `variables` here (and this edit should apply to all of the descriptions below):

```suggestion
        site must no longer be using the classic environment variables experience. Migrate now with the Netlify UI.
```

Otherwise this looks good for now 👍  

Curious question, can we include links in these docs? If so, it might be helpful to link to the classic page, so they know what we're talking about. Once it's published, the link will be https://docs.netlify.com/environment-variables/classic-experience/.  If it's not possible, then that's ok.

Something like this?

```suggestion
        site must no longer be using the <a href="https://docs.netlify.com/environment-variables/classic-experience/">classic environment variables experience</a>. Migrate now with the Netlify UI.
```